### PR TITLE
Use @site to maintain compatibility with Jekyll 3.5

### PR DESCRIPTION
**Pull request by benbalter**
See https://github.com/jekyll/jekyll/issues/6154.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,13 +6,14 @@ cache: bundler
...
+    - rvm: 2.4.0
+      env: JEKYLL_VERSION="~> 3.5"
+    - rvm: 2.4.0
       env: GH_PAGES=true
   allow_failures:
     - env: GH_PAGES=true # Jekyll 2.4 will fail tests
   fast_finish: true
-
+  
 rvm:
```
Comment (ashmaroli) [inline .travis.yml]: xtra whitespace here..

**Comment by antoniocosta** [comment]
Tested https://github.com/jekyll/jekyll-redirect-from/pull/157/commits/bc16f3fe5cf95bf1b76dc4cb487c70348025c98f with Jekyll 3.5. 
It solves https://github.com/jekyll/jekyll/issues/6154

**Comment by benbalter** [comment]
@ashmaroli can this be closed now that https://github.com/jekyll/jekyll/pull/6163 merged?

**Comment by ashmaroli** [comment]
Yes. This is no longer required.

**Pull request closed without merging**